inductiva/templates/protein_solvation/gromacs/charge_computation.json.jinja

```diff
@@ -0,0 +1,5 @@
+[{
```

Comment:
I think we should follow the approach below of uniformization when calling the protein, but this is ok for now!